src/CONST.ts

```diff
@@ -13,6 +13,7 @@ import type {Unit} from './types/onyx/Policy';
 type RateAndUnit = {
     unit: Unit;
     rate: number;
+    currency: string;
```

Comment:
NAB: IMO let's delete this type and just use `MileageRate`, since `RateAndUnit` is a subset of it, and the name is not great and now inaccurate. It can be cleaned up in a follow up.